boards/nordic/nrf52840dongle/Kconfig

```diff
@@ -3,13 +3,12 @@
 # Copyright (c) 2018-2023 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-if BOARD_NRF52840DONGLE
+config BOARD_NRF52840DONGLE
+	select BOARD_EARLY_INIT_HOOK
 
 config BOARD_HAS_NRF5_BOOTLOADER
 	bool "Board has nRF5 bootloader"
 	default y
```

Comment:
Shouldn't `depends on BOARD_NRF52840DONGLE` be added here?

Reply:
Not needed, there is only one board and in hwmv2 this file is only loaded when this board is used